safe/gui/tools/wizard/step_kw33_multi_classifications.py

```diff
@@ -0,0 +1,1027 @@
+            for i, hazard_classification in enumerate(hazard_classifications):
+                exposure_combo_box.addItem(hazard_classification['name'])
+                exposure_combo_box.setItemData(
+                    i + 1, hazard_classification, Qt.UserRole)
+                if self.layer_mode == layer_mode_continuous:
+                    current_hazard_classifications = self.thresholds.get(
+                        exposure['key'])
+                else:
+                    current_hazard_classifications = self.value_maps.get(
+                        exposure['key'])
+                if current_hazard_classifications:
+                    current_hazard_classification = \
+                        current_hazard_classifications.get(
+                            hazard_classification['key'])
+                    if current_hazard_classification:
+                        is_active = current_hazard_classification.get('active')
+                        if is_active:
+                            current_index = i + 1
+            # Set current classification
+            exposure_combo_box.setCurrentIndex(current_index)
+
+            # Add edit button
+            exposure_edit_button = QPushButton(tr('Edit'))
+            exposure_edit_button.clicked.connect(
+                partial(self.edit_button_clicked,
+                        edit_button=exposure_edit_button,
+                        exposure_combo_box=exposure_combo_box,
+                        exposure=exposure))
+            if current_index == 0:
+                # Disable if there is no classification chosen.
+                exposure_edit_button.setEnabled(False)
+
+            exposure_combo_box.currentIndexChanged.connect(
+                partial(
+                    self.classifications_combo_box_changed,
+                    exposure=exposure,
+                    exposure_combo_box=exposure_combo_box,
+                    edit_button=exposure_edit_button
+                    )
+            )
+
+            # Arrange in layout
+            exposure_layout.addWidget(exposure_label)
+            exposure_layout.addWidget(exposure_combo_box)
+            exposure_layout.addWidget(exposure_edit_button)
+
+            # Stretch
+            exposure_layout.setStretch(0, 0)
+            exposure_layout.setStretch(0, 0)
+            exposure_layout.setStretch(0, 1)
+            self.left_layout.addLayout(exposure_layout)
+
+            self.left_layout.addStretch(1)
+
+            # Adding to step's attribute
+            self.exposures.append(exposure)
+            self.exposure_combo_boxes.append(exposure_combo_box)
+            self.exposure_edit_buttons.append(exposure_edit_button)
+            self.exposure_labels.append(label)
+
+    def edit_button_clicked(self, edit_button, exposure_combo_box, exposure):
+        """Method to handle edit button."""
+        LOGGER.debug(exposure['key'])
+        LOGGER.debug(self.get_classification(exposure_combo_box)['key'])
```

Comment:
I usually try to give some context when we use the logger. Because then in the console, we see only some meaningless message.